pkgs/development/libraries/eclib/default.nix

```diff
@@ -14,16 +14,14 @@ assert withFlint -> flint != null;
 
 stdenv.mkDerivation rec {
   pname = "eclib";
-  version = "20190909"; # upgrade might break the sage interface
+  version = "20210503"; # upgrade might break the sage interface
   # sage tests to run:
   # src/sage/interfaces/mwrank.py
   # src/sage/libs/eclib
   # ping @timokau for more info
-  src = fetchFromGitHub {
-    owner = "JohnCremona";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "0y1vdi4120gdw56gg2dn3wh625yr9wpyk3wpbsd25w4lv83qq5da";
+  src = fetchurl {
+    url = "https://github.com/JohnCremona/eclib/releases/download/${version}/eclib-${version}.tar.bz2";
+    sha256 = "sha256-cCm44EsrQ0mUA+E50XVt6E7Oz5lXhyZTI82w2QHaBqs=";
```

Comment:
Is there a reason for this change?

Reply:
Thanks for the review! I think I'll ping again you once this reaches a Sage beta.

The main reason for the change is that v20210503 is not tagged. Combined with the fact that eclib release tarballs are now manually uploaded by the developer (no longer just a repository snapshot automatically created by GitHub), it looked like the most authoritative source.

Reply:
> I think I'll ping again you once this reaches a Sage beta.

:+1: 

> The main reason for the change is that v20210503 is not tagged. Combined with the fact that eclib release tarballs are now manually uploaded by the developer (no longer just a repository snapshot automatically created by GitHub), it looked like the most authoritative source.

I see, thank you for clarifying. It would be nice to add this information to the package declaration. I like to put information like "releases not tagged, update hash on `src` on update" near the `version` attribute where it is easy to spot when someone tries to update (and visible in the diff when reviewing and update).

A comment about the manually uploaded release tarballs would also be nice to have.

Reply:
Why can't we ping upstream that they start to tag the releases again?